jetty-integrations/jetty-infinispan/jetty-infinispan-remote-query/pom.xml

```diff
@@ -9,7 +9,6 @@
   <name>Integrations :: Infinispan :: Remote with Querying</name>
   <properties>
     <bundle-symbolic-name>${project.groupId}.infinispan.remote.query</bundle-symbolic-name>
-    <infinispan.docker.image.version>11.0.9.Final</infinispan.docker.image.version>
```

Comment:
where's the version of infinispan image configured now, I can't find it?

Reply:
top pom (https://github.com/eclipse/jetty.project/pull/9782/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R200). I even keep a comment maybe we can remove the comment now? :)